# The file name to which we want to save the point cloud
string filename
---
bool success
